examples.py

```diff
@@ -20,10 +18,10 @@ def make_application(**settings):
     application = web.Application([web.url(r'/', SimpleHandler)], **settings)
     content.set_default_content_type(application, 'application/json',
                                      encoding='utf-8')
-    content.add_binary_content_type(application, 'application/msgpack',
-                                    msgpack.packb, msgpack.unpackb)
-    content.add_text_content_type(application, 'application/json', 'utf-8',
-                                  json.dumps, json.loads)
+    content.add_transcoder(application, 'application/msgpack',
+                           transcoders.MsgPackTranscoder())
```

Comment:
since you're going this far, why not remove the mime type and have the ``add_transcoder`` method read a constant from the transcoder object?